extensions-core/s3-extensions/src/test/java/org/apache/druid/storage/s3/output/RetryableS3OutputStreamTest.java

```diff
@@ -171,6 +171,26 @@ public void testWriteSmallBufferShouldSucceed() throws IOException
     s3.assertCompleted(chunkSize, 600);
   }
 
+  @Test
+  public void testWriteSmallBufferExactChunkSizeShouldSucceed() throws IOException
+  {
+    chunkSize = 128;
+    final int fileSize = 128 * 5;
+    try (RetryableS3OutputStream out = new RetryableS3OutputStream(
+        config,
+        s3,
+        path,
+        s3UploadManager
+    )) {
```

Comment:
Nit: More readable like this:
```suggestion
    try (RetryableS3OutputStream out =
              new RetryableS3OutputStream(config, s3, path, s3UploadManager)) {
```

Reply:
Done.